src/xcode/ENA/ENA/Source/Services/Event Registration/QRCodePosterTemplate/QRCodePosterTemplateProvider.swift

```diff
@@ -30,6 +30,17 @@ class QRCodePosterTemplateProvider: QRCodePosterTemplateProviding {
 			.eraseToAnyPublisher()
 	}
 
+	func defaultQRCodePosterTemplate() -> SAP_Internal_Pt_QRCodePosterTemplateIOS {
+		guard
+			let url = Bundle(for: type(of: self)).url(forResource: "qr_code_template", withExtension: "bin"),
+			let data = try? Data(contentsOf: url),
+			let template = try? SAP_Internal_Pt_QRCodePosterTemplateIOS(serializedData: data)
+		else {
+			fatalError("Cannot load default QR code poster template")
```

Comment:
A fatalError looks a bit hard here. If for some reason the template can't get loaded the app will crash? I don't think we should do that here.
Better catch the errors and show en alert, write a log entry or at least do nothing ?

Reply:
Fix